### DISCUSSION TITLE
Just another Muse...

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1217] [Team Posts]
Ever heard of that?

https://mu.se/

**Comment by spamatica** [comment]
Oh! That is new. Not a sequencer at least.

Nice domain name, kinda kicking myself for never checking for it.

**Comment by spamatica** [comment]
It says "(Acquired...) after Audacity and MusEScore, what the hell? They can't possibly own them!?

**Comment by kybos** [comment]
I'm not sure myself what's that supposed to mean... Quite strange. The Audacity "take-over" is very new, from yesterday I think.

**Comment by kybos** [comment]
https://www.musictech.net/news/industry/audacity-acquired-muse-group-ultimate-guitar-musescore/

**Comment by spamatica** [comment]
Okay, name aside, very weird.
I mean, I suppose it is possible to by the rights of open source software too, but as most old OSS they likely have a rather large amount of contributors. Now, if they aren't changing the license it doesn't really matter, but then what have they bought? 
Just speculating, maybe there were companies created of the development team for both musescore and audacity and they bought those?

**Comment by kybos** [comment]
I'm no expert, it's difficult to understand how somebody can "acquire" a FOSS project which is not owned by any legal subject, but by the community? And of course they can't change the GPL license to something lesser. But they can make money with related services (like they do with the MuseScore cloud), that's not illegal.

**Comment by terminator356** [comment]
This sucks. Seems that Muse Score is now a 'product' with a 'free trial'.
I expect that we'll soon be receiving a 'cease and desist' order from lawyers regarding our name MusE.
Don't laugh, it could happen, no?
I hate Ultimate Guitar. I'll tell you why some day.

**Comment by kybos** [comment]
> I expect that we'll soon be receiving a 'cease and desist' order from lawyers regarding our name MusE.

I confess that was my first thought when I saw their web page. Nothing to laugh about I'm afraid, wouldn't be the first such case...

**Comment by spamatica** [comment]
Musescore download still exists at https://musescore.org/ (as opposed to musescore.com), it's just that they try to hide it... 

Sigh... these times we live in. Remember when the internet was the solution to really setting people free? Free knowledge, free thought. That came and went real fast...

**Comment by spamatica** [comment]
Oh, how ominous, a question on the forum about MusEs future :D 
Though I think the intentions of this question is benign.

**Comment by terminator356** [comment]
On the other side of things, maybe we will soon receive some kind of offer.
That post on the forum seems odd... "I am willing to help but...". Help how exactly? Coding?

You may find this strange, but yesterday a voice came to me and said
 "wouldn't it be cool if a hardware manufacturer decided to include MusE in their product?"
Where upon I imagined our little project on a fancy new keyboard's screen.

Tim the entrepreneur speaking:
How would you guys like to form an actual company?
I mean go at it full time.
Finally make a Windows version to attract much more attention.
It's the only way anyone will take us seriously. If it runs on Windows, people will be more confident in it.
Sell MusE with extra paid features.
Collaboration features with subscriptions etc.
Or get a sponsor. Maybe a GitHub sponsor.
Regardless, we really ought to finally compile a Windows version and make it available.
I think it is about time we tried.

**Comment by terminator356** [comment]
That post on the forum came from someone who just joined LM today (May 4).
Hmm.....
And... May the Fourth be with you !   ;-)

**Comment by kybos** [comment]
Just noticed - that guy sent me a PM this morning already...

**Comment by spamatica** [comment]
Hehe, yeah I saw the guy just joined. Saviour or interloper? :D

I would think it's probably a normal guy that just explained himself a bit awkwardly ;)

Regarding hardware manufacturer. Do you remember the italian linux-running keyboard, Lion-something? As I recall they used MusE as the internal midi sequencer. Hmm, could be I remember it wrong.

About making a company, I'm not against others trying but I'm probably not interested. High stress work got the best of me in the past and I really don't want to do that again.

Making ports of MusE does interest me though. I have tried to make both a OSX (which nearly worked) and a Windows port (which was not close). But I'm not sure it interests me enough though...
The benefit would be the huge user-base on Windows... But the question is, would they come? There are a huge amount of DAWs on Windows... Admittedly very few that are open source....

**Comment by spamatica** [comment]
> Just noticed - that guy sent me a PM this morning already...

Oh he did? Did he write something else?

Regardless, we probably should not promise too much. Better if it happens a bit organically. Unless he wants to fund development, that is another thing.

**Comment by terminator356** [comment]
Hmm. The plot thickens.
I would like to respond to that post but I'll wait for your input.
Perhaps tread carefully here. He (she) could be a major player.
Do you want to start on that rough roadmap you were just talking about?

The only thing I can say is that we are coders **highly dedicated** to the love of this project,
 with Robert having ~20 years, me ~15 years, and kybos... well the equivalent of 10 years.
With a handful of other dedicated contributors over the years.
I could outline my personal tasks/goals - everything to do with the engines, performance, time stretching,
 adding support for new technologies like VST3 etc.

**Comment by kybos** [comment]
> Just noticed - that guy sent me a PM this morning already...
> Oh he did? Did he write something else?
> 
No, same text as in forum.

**Comment by terminator356** [comment]
> Do you remember the italian linux-running keyboard, Lion-something

Found it. LionsTracs.
